src/tgp.cpp

```diff
@@ -455,6 +455,66 @@ static int *HeightMapMakeHistogram(Height h_min, [[maybe_unused]] Height h_max,
 	return hist;
 }
 
+/**
+ * Adjust the landscape to create lowlands on average (tropic landscape).
+ * @param fheight The height to adjust.
+ */
+static void SineTransformLowlands(double &fheight)
```

Comment:
an in/out parameter (not correctly marked as such in doxygen, incidentally) with no return value is mildly odd - any particular reason for not just returning the (new) value?